log4j-mongodb/pom.xml

```diff
@@ -122,8 +122,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <!-- TODO: fix concurrent download of MongoDB distribution. -->
-          <forkCount>1</forkCount>
+          <skip>true</skip>
```

Comment:
Previously Surefire didn't run in parallel to prevent the download of MongoDB multiple times.
Now we don't have any unit tests any more, so we can skip Surefire.